sklearn/linear_model/tests/test_base.py

```diff
@@ -53,6 +53,30 @@ def test_linear_regression():
     assert_array_almost_equal(reg.predict(X), [0])
 
 
+def test_linear_regression_cholesky():
```

Comment:
Would it be better to parameterise the existing tests with the different values of `solver` than to introduce new tests?

Reply:
Yes! Changed this.